dvc/cli.py

```diff
@@ -117,6 +161,83 @@ def parse_args(self, args=None, namespace=None):
             self.error(msg % " ".join(argv), getattr(args, "func", None))
         return args
 
+    def get_cmd_choices(self):
```

Comment:
no need to keep it public
```suggestion
    def _get_cmd_choices(self):
```